tools/build_test.py

```diff
@@ -0,0 +1,100 @@
+import sys
+from os.path import join, abspath, dirname, exists, basename
+ROOT = abspath(join(dirname(__file__), ".."))
+sys.path.insert(0, ROOT)
+
+import argparse
+
+from export import EXPORTERS
```

Comment:
I copied the style from project.py. I will move the standard library imports above the relative import fix.